Asterisk.2013/Asterisk.NET/Manager/Originate.cs

```diff
@@ -141,22 +141,6 @@ public long Timeout
 
         #endregion
 
-        #region Variable
-
-        /// <summary>
-        ///     Get/Set the variables to set on the originated call.<br />
-        ///     Variable assignments are of the form "VARNAME=VALUE". You can specify
-        ///     multiple variable assignments separated by the '|' character.<br />
-        ///     Example: "VAR1=abc|VAR2=def" sets the channel variables VAR1 to "abc" and VAR2 to "def".
-        /// </summary>
-        public string Variable

```

Comment:
I'd like if someone would double-check this, but it seems like the Originate class itself isn't actually used anywhere.  I commented out the entire file, and the solution still builds without any problems.  It looks like all the functionality already exists in OriginateAction - I think we should consider just deleting the file entirely.